stdlib/importlib/metadata/__init__.pyi

```diff
@@ -41,6 +41,9 @@ class _EntryPointBase(NamedTuple):
 
 class EntryPoint(_EntryPointBase):
     pattern: ClassVar[Pattern[str]]
+    if sys.version_info >= (3, 11):
+        def __init__(self, name: str, value: str, group: str) -> None: ...
+
```

Comment:
The allowlist entry had a comment saying that it was to do with `Protocol.__init__`, so I thought I could safely remove the allowlist entry now that we were bumping to mypy 0.981. Actually the comment was wrong, the allowlist entry had nothing to do with `Protocol.__init__`. But I figured fixing the allowlist entry was just as easy as correcting the comment next to the entry.